### ISSUE TITLE
Not working on ubuntu No Request, No Paramter

### ISSUE THREAD

**Issue by JinD3vi1** [zecure/shadowd#33]
I have successfully configured Shadow Daemon on ubuntu running apache2 server but its not blocking anything after enabling blacklist. No parameter no requests record on UI. Kindly help me out. I have already tested connector file and no error from it. I have given wrong profile ID to test if its assessing my connector file or not but the error it throw showed that it is assessing it and properly configured. The only issue is right now i have added some rules in my blacklist but no blockage/request recorded. Kindly help me out in it. Related screenshot attached.

<img width="1395" alt="Screenshot 2019-10-24 at 1 56 49 PM" src="https://user-images.githubusercontent.com/56957227/67472452-90558200-f66a-11e9-9791-01e2509f9604.png">
<img width="866" alt="Screenshot 2019-10-24 at 2 27 50 PM" src="https://user-images.githubusercontent.com/56957227/67472455-90558200-f66a-11e9-86e3-7bf5dd7fddf2.png">
<img width="1331" alt="Screenshot 2019-10-24 at 2 28 01 PM" src="https://user-images.githubusercontent.com/56957227/67472456-90ee1880-f66a-11e9-837b-2fc1638ebe17.png">
<img width="1440" alt="Screenshot 2019-10-24 at 2 28 17 PM" src="https://user-images.githubusercontent.com/56957227/67472457-90ee1880-f66a-11e9-8dcc-f6bcf915b35d.png">

**Comment by zit-hb** [comment]
Hey. You can try to enable the learning mode of the profile. Then every request is recorded and you can have a look at impacts of certain parameters etc.

Also, make sure that "observe" is not enabled for the connector. This option disables all protections.

**Comment by JinD3vi1** [comment]
Hey ! I have tried what you suggested but it ends with no luck. Can you help me further with all this? I am attaching screenshot of phpinfo() and Coonector.ini file. Do also guide me where logs of Shadow Daemon recorded as there is no file in /var/logs/shadow. 
<img width="945" alt="Screenshot 2019-10-28 at 12 05 01 PM" src="https://user-images.githubusercontent.com/56957227/67658672-124fee80-f97c-11e9-9403-73b2a6d26ef4.png">
[connectorsini.txt](https://github.com/zecure/shadowd/files/3777510/connectorsini.txt)

**Comment by JinD3vi1** [comment]
Mistakenly closed.

**Comment by zit-hb** [comment]
Hey, sorry about the delay. I only saw the notification about the closing of the issue but not the message before that. There are multiple positions you can enable logging. For one, in the [connector](https://shadowd.zecure.org/documentation/connectors/#debug:36331316c021a51589fe614a3f875457). You can find the messages in the PHP or web server log file, depending on your web server configuration. Can you share your connector config here?
Second, you can enable logging for shadowd. You have to edit the docker-compose.yml and override the [command](https://docs.docker.com/compose/compose-file/#command) of shadowd. You can use `--verbose` and `--log <file>` to get a lot of detailed information. Be aware though that the log file is written inside of the Docker container. You can get into the container by running `./shadowdctl exec shadowd bash`.

**Comment by figassis** [comment]
Logging not working, I've tried adding the flags:

command: ["sh", "-c","/docker-entrypoint.sh /usr/bin/shadowd -c /etc/shadowd/shadowd.ini -U shadowd -G shadowd -L /var/log/shadowd.log -V"]

no /var/log/shadowd.log is every created, I also tried creating one, and its always empty

I set the connector.ini to debug and observe, also set the profile to learning and passive, nothing works.

I deployed shadowd on kubernetes and connecting to it always hangs, no response, but the connection does not get refused which would happen if the port was incorrect. Locally on docker it works, still no logs, but works.

Can anyone help?
